### PR TITLE
Make instructions consistent with current  structure

### PR DESCRIPTION
**Pull request by ebeasant-arm**
Correct the build instructions to match the current repository names and locations.

### CODE REVIEW

**Review by dominic-mulligan-arm — DISMISSED** [review]
Looks good to me.

File: BUILD_INSTRUCTIONS.markdown [hunk 1]
```diff
@@ -16,10 +16,16 @@ token after creating an account here:
...
+```
+git clone git@github.com:veracruz-project/veracruz.git
+export VERACRUZ_ROOT=$PWD/veracruz
+```
 Once you have a local copy of the Veracruz source and your token:
 
 ```
-cd SGXRustOPTEEDevelDocker
+cd veracruz-docker-image
 make VERACRUZ_ROOT=<path to your veracruz directory> IAS_TOKEN=<your Intel Attestation Service token>
```
Comment (dominic-mulligan-arm) [inline BUILD_INSTRUCTIONS.markdown:29]: Is the `VERACRUZ_ROOT=<path to your Veracruz directory>` part required, given the `export` that you added above?

Comment (ebeasant-arm) [reply BUILD_INSTRUCTIONS.markdown:29]: Good point - Will update...

File: BUILD_INSTRUCTIONS.markdown [hunk 2]
```diff
@@ -16,11 +16,17 @@ token after creating an account here:
 
 https://api.portal.trustedservices.intel.com/EPID-attestation
 
+Clone the Veracruz repository:
+
+```
+git clone git@github.com:veracruz-project/veracruz.git
```
Comment (dreemkiller) [inline BUILD_INSTRUCTIONS.markdown:22]: Doesn't this need to be a recursive clone?

File: BUILD_INSTRUCTIONS.markdown [hunk 3]
```diff
@@ -16,11 +16,17 @@ token after creating an account here:
...
+export VERACRUZ_ROOT=$PWD/veracruz
+```
 Once you have a local copy of the Veracruz source and your token:
 
 ```
-cd SGXRustOPTEEDevelDocker
-make VERACRUZ_ROOT=<path to your veracruz directory> IAS_TOKEN=<your Intel Attestation Service token>
+cd veracruz-docker-image
+make IAS_TOKEN=<your Intel Attestation Service token>
```
Comment (dreemkiller) [inline BUILD_INSTRUCTIONS.markdown:29]: Don't we also need to set VERACRUZ_ROOT?

Comment (ebeasant-arm) [reply BUILD_INSTRUCTIONS.markdown:29]: VERACRUZ_ROOT is set in a previous step after the repo clone : I could add some more detail explaining that ?

Comment (dreemkiller) [reply BUILD_INSTRUCTIONS.markdown:29]: Doesn't the make command itself need to hae VERACRUZ_ROOT=...,, or does an environment variable work? (I can't recall)

Comment (dominic-mulligan-arm) [reply BUILD_INSTRUCTIONS.markdown:29]: @dreemkiller: `make` mirrors environment variables from the environment within which it is invoked as implicit make variables.

Comment (dreemkiller) [reply BUILD_INSTRUCTIONS.markdown:29]: cool.

File: BUILD_INSTRUCTIONS.markdown (hunk 2 above)
Comment (dominic-mulligan-arm) [reply BUILD_INSTRUCTIONS.markdown:22]: Yes, I think so.

**Comment by dominic-mulligan-arm** [comment]
@ebeasant-arm, when you have time, can you update this pull request so we can get it merged?

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 5dc7825445e1fd5e5095ea068736fde03c22ee5a
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=80323ff0-48fc-4c5a-b2d6-c43076b99d4c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 1441841b22444f9e533d4beec71c36def22b418d
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=758d1694-57ad-4b93-9bf5-2ab4056e9a0c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 808f782dbfcb9d4e64302605405a7c48dc6c0b6c
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=7a6d08bd-7f9b-4735-8861-6db165cd6c68%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by dominic-mulligan-arm — APPROVED** [review]

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 9ec5fd73a1a8631b23a226996d3ac14486ee95b4
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=58a5fa37-8ae5-4dc0-bce6-8cab47259caf%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by dominic-mulligan-arm** [comment]
+1 reached, merging.

**Pull request merged**